.circleci/config.yml

```diff
@@ -344,6 +355,13 @@ jobs:
                 postgres_password: SANDBOX_POSTGRES_PASSWORD
                 postgres_username: SANDBOX_POSTGRES_USERNAME
                 session_secret: SANDBOX_SESSION_SECRET
+            - run:
```

Comment:
:+1: I like clearing the DB here because if someone accidentally gets it to run for `prod` or `staging` the `SANDBOX_*` env variables should still point to sandbox.